lib/main.js

```diff
@@ -159,9 +173,25 @@ function loadScript(filename) {
 const SCRIPT_CACHE = {
 	cf: loadScript(`${__dirname}/contextify.js`).wrap('(function(require, host) { ', '\n})')._compileVM(),
 	sb: loadScript(`${__dirname}/sandbox.js`).wrap('(function (vm, host, Contextify, Decontextify, Buffer) { ', '\n})')._compileVM(),
-	exp: new VMScript('({exports: {}})')._compileVM()
+	exp: new VMScript('({exports: {}})')._compileVM(),
+	runTimeout: new VMScript('fn()', 'timeout_bridge.js')._compileVM()
 };
 
+const TIMEOUT_CONTEXT = vm.createContext();
```

Comment:
I'd suggest changing that to `const TIMEOUT_CONTEXT = null` and moving `vm.createContext()` inside the `doWithTimeout` function. Just a performance tweak for situations when timeout is not used.